testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/beanvalidation/hibernate/validator/BootStrapValidationTestCase.java

```diff
@@ -56,6 +57,8 @@
     public static Archive<?> deploy() {
         final WebArchive war = ShrinkWrap.create(WebArchive.class, "testbootstrapvalidation.war");
         war.addPackage(BootStrapValidationTestCase.class.getPackage());
+        // Hibernate Validator needs the following runtime permission
+        war.addAsManifestResource(createPermissionsXmlAsset(new RuntimePermission("accessDeclaredMembers")), "permissions.xml");
```

Comment:
I think it is just a case of this test-case which invokes Hibernate Validators directly. Correct me if I am wrong. Calling them through an integration layer should work fine. It is a known issue that Hibernate Validator needs permissions.